### PR TITLE
Update bucks email colours

### PR DESCRIPTION
**Pull request by struan**
Allow email header image filename to be customised.
Update bucks email branding.
Fixes https://github.com/mysociety/fixmystreet-commercial/issues/1272

Please check the following:

- [x] All cobrand-specific commits start their commit message with the cobrand in square brackets;
- [x] Have you updated the changelog? If this is not necessary, put square brackets around this: skip changelog

### CODE REVIEW

**Comment by struan** [comment]
Fixes mysociety/fixmystreet-commercial#1272

File: templates/email/default/_email_settings.html [hunk 1]
```diff
@@ -86,6 +86,7 @@
 start_padded_box = '<table cellspacing="0" cellpadding="' _ column_padding _ '" border="0" width="100%"><tr><th style="' _ td_style _ '">'
 end_padded_box = '</th></tr></table>'
 
+logo_image = "images/email-logo.gif"
```
[the remark this answers is not in the document]
Comment (struan) [reply templates/email/default/_email_settings.html]: I didn't really want to go back and fix all the img paths everywhere to be images as part of this. We can possibly make that a ticket to do at some point later though.

[the remark this answers is not in the document]
Comment (dracos) [reply templates/email/default/_email_settings.html]: The only img paths are the ones added in this PR (so two files)? I wasn't saying you needed to change every web directory of images at this point (especially now this needs rebasing on top of the similar change already made). northamptonshire already has both, for example :)

[the remark this answers is not in the document]
Comment (struan) [reply templates/email/default/_email_settings.html]: Oh, right, I see, just be fine with some cobrands having img/ and images/ for now and then later on tidy it up to be consistent.

File: templates/email/default/_email_settings.html [hunk 2]
```diff
@@ -25,6 +25,7 @@
 header_text_color = color_black
 header_padding = "15px 20px" # a full CSS padding property (eg: top/right/bottom/left)
 
+logo_file = "email-logo.gif"
```
Comment (dracos) [inline templates/email/default/_email_settings.html]: You can remove the `DEFAULT logo_file = 'email-logo.gif';` line from `_email_top.html` then.

**Comment by dracos** [comment]
Oh sorry there's stil the issue of submit.html (which is used in Bristol) has a footer with white on white writing. "might be easiest to have empty templates/email/{cobrand}/_submit_footer.html templates, or come up with a way to change the footer text colour."

**Comment by struan** [comment]
merged in 42004232f

**Pull request closed without merging**